book/src/pickers.md

```diff
@@ -0,0 +1,11 @@
+## Using pickers
+
+Helix has a variety of pickers, which are interactive windows used to select various kinds of items. These include a file picker, global search picker, and more. Most pickers are accessed via keybindings in [space mode](./keymap.md#space-mode). Pickers have their own [keymap](./keymap.md#picker) for navigation.
+
+### Filtering Picker Results
+
+Most pickers perform fuzzy matching using [fzf syntax](https://github.com/junegunn/fzf?tab=readme-ov-file#search-syntax). Two exceptions are the global search picker, which uses regex, and the workspace symbol picker, which passes search terms to the LSP.
+
+If a picker shows multiple columns, you may apply the filter to a specific column by prefixing the column name with `%`. Column names can be shortened to any prefix, so `%p`, `%pa` or `%pat` all mean the same as `%path`. For example, `helix %p .toml$ !lang` searches for the term "helix" within files with paths ending in ".toml" but not including "lang".
+
+Note that the [current selection register](./registers.md#special-registers) can be used to insert the text of the current selection using `Ctrl-r`-`.`. In addition, the global search picker will use the contents of the [search register](./registers.md#special-registers) if you press `Enter` without typing a filter. For example, pressing `*`-`Space-/`-`Enter` will start a global search for the currently selected text.
```

Comment:
Good idea talking about special registers here. More generically you can insert the contents of any register with `C-r` and then the register name. A common case that comes up here or with `:open` is to use `<C-r>%` to insert the current file's path, and then you can use `C-w` to chop off the last path segment.

So in this block let's say that you can insert any register contents and then show examples (`<C-r>.` and `<C-r>%`) in the next sentences.

Reply:
```suggestion
Note that the [current selection register](./registers.md#special-registers) can be used to insert the text of the current selection using `Ctrl-r`-`.`. In addition, the global search picker will use the contents of the [search register](./registers.md#default-registers) if you press `Enter` without typing a filter. For example, pressing `*`-`Space-/`-`Enter` will start a global search for the currently selected text.
```

one small thing here - `/` is a regular register, it's used as the default for search operations

Reply:
Looks like many of the other keybinds from Prompt are supported in Picker, not just `Ctrl-R`. Full list below. These keybindings and the usage example could be documented in `keymap.md` instead of `pickers.md` as it could also be useful for Prompt.
|keybind(s)|action|
|-|-|
|Alt-b, Ctrl-Left|Backward a word|
|Ctrl-b, Left|Backward a char|
|Alt-f, Ctrl-Right|Forward a word|
|Ctrl-f, Right|Forward a char|
|Ctrl-e, ~~End~~|Move prompt end|
|Ctrl-a, ~~Home~~|Move prompt start|
|Ctrl-w, Alt-Backspace, Ctrl-Backspace|Delete previous word|
|Alt-d, Alt-Delete, Ctrl-Delete|Delete next word|
|~~Ctrl-u~~|~~Delete to start of line~~|
|Ctrl-k|Delete to end of line|
|Backspace, Ctrl-h, Shift-Backspace|Delete previous char|
|Delete, ~~Ctrl-d~~|Delete next char|
|~~Ctrl-s~~|~~Insert a word under doc cursor, may be changed to Ctrl-r Ctrl-w later~~|
|Ctrl-p, Up|Select previous history|
|Ctrl-n, Down|Select next history|
|Ctrl-r|Insert the content of the register selected by following input char|
|~~Tab~~|~~Select next completion item~~|
|~~BackTab~~|~~Select previous completion item~~|
|Enter|Open selected|

Reply:
I could add these to the `picker` keymap section, or I could just say "Prompt keybindings are supported in pickers except where they conflict with picker keybindings.

There's a few exceptions like `ctrl-u`, but I think those could probably be implemented in pickers?

Reply:
> I could add these to the `picker` keymap section, or I could just say "Prompt keybindings are supported in pickers except where they conflict with picker keybindings.

Sounds good from my perspective.